mimalloc-sys-test/Cargo.toml

```diff
@@ -3,16 +3,10 @@ name = "mimalloc-sys-test"
 version = "0.1.0"
 authors = ["gnzlbg <gonzalobg88@gmail.com>"]
 edition = "2015"
-build = "build.rs"
 
```

Comment:
Please re-add the `build = "build.rs"` here.

Reply:
I dropped it because flate2-rs need it and the [build-scripts](https://doc.rust-lang.org/cargo/reference/build-scripts.html) docs say it's optional when a script is "build.rs".  (_By default Cargo looks up for "build.rs" file in a package root (even if you do not specify a value for build)._)

 I ran a rustfmt and thought it auto-deleted it, but I guess that was me.  I'll revert this.

Reply:
Yes, i just find it clearer when the Cargo.toml shows that there is a build.rs.